src/google/protobuf/compiler/objectivec/import_writer.h

```diff
@@ -55,6 +55,7 @@ class ImportWriter {
 
   void AddFile(const FileDescriptor* file, const std::string& header_extension);
   void AddRuntimeImport(const std::string& header_name);
+  std::string ModuleForFile(const FileDescriptor* file);
```

Comment:
Add a comment for this saying it will return the empty string is there is no module for the file.

And I guess it should say it doesn't handle bundled proto files.

Reply:
on it

Reply:
done